PrebidMobile/PrebidMobile-core/src/main/java/org/prebid/mobile/Util.java

```diff
@@ -399,6 +430,8 @@ static void apply(HashMap<String, String> bids, Object adObj) {
         if (adObj.getClass() == getClassFromString(MOPUB_BANNER_VIEW_CLASS)
                 || adObj.getClass() == getClassFromString(MOPUB_INTERSTITIAL_CLASS)) {
             handleMoPubKeywordsUpdate(bids, adObj);
+        } else if (adObj.getClass() == getClassFromString(MOPUB_NATIVE_CLASS)) {
```

Comment:
What if we use RequestParameters.Builder() as the MoPub native class like GAM? Would it work?

Reply:
Done.